pkg/cmd/secrets.go

```diff
@@ -620,6 +631,7 @@ func init() {
 	secretsDownloadCmd.Flags().StringP("project", "p", "", "project (e.g. backend)")
 	secretsDownloadCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
 	secretsDownloadCmd.Flags().String("format", models.JSON.String(), fmt.Sprintf("output format. one of %s", validFormatList))
+	secretsDownloadCmd.Flags().StringP("name-transformer", "n", "", fmt.Sprintf("(BETA) output name transformer. one of %v", validNameTransformersList))
```

Comment:
We should hold off on adding the `-n` short flag since we may want to use that for something else (espec. something global).

Reply:
Could we pick a different short flag? I'm a little worried that `--name-transformer` is just way too long to be useful.

Some alternatives:

- `-x` for a "cross" mnemonic
- `-r` for a "rename" mnemonic

Reply:
Another alternative could be to drop the short flag and make the "long flag" something shorter like `--rename`

Reply:
Do we expect users to be running this ad hoc or will this mostly be in scripts?

Reply:
I'm not sure yet but the biggest use case I'm thinking of right now is Terraform:

```
doppler run --name-transformer tf-var -- terraform apply
# vs
doppler run -x tf-var -- terraform apply
```

Reply:
A short flag can always be added later, but once added it can't be removed. I think we should ship without one for now and get some usage before adding it.

Reply:
Copy that! I'll remove the short-flag for now